okapi-core/src/main/raml/RoutingEntry.json

```diff
@@ -68,6 +68,10 @@
       "items": {
         "type": "string"
       }
+    },
+    "delegateCORS": {
+      "description": "Okapi handles CORS by default. Set to true to delegate CORS handling to the module",
```

Comment:
consider mentioning that it's only done when using /_/invoke/tenant/<tid>/<path>

Reply:
Added.